### PR TITLE
fix: possible config example

### PR DESCRIPTION
**Pull request by arthur-observe**
Add a new values-minimal.yaml chart

Idea: Maybe we just create a new example and then point to this as a good default but not interfere with any existing?

### CODE REVIEW

File: examples/stack/values-minimal.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        memory: 128Mi
+    config:
+      mem_buf_limit: 10MB
+
+metrics:
+  grafana-agent:
+    prom_config:
+    scrape_interval: 60s
+    scrape_configs:
+      cadvisor_action: drop
```
Comment (NathanNam) [inline examples/stack/values-minimal.yaml:35]: I'm fine with this approach.

Why did we set `cadvisor_action: drop`?

https://observeinc.slack.com/archives/C06EBP24T1V/p1706044118395189

Comment (NathanNam) [reply examples/stack/values-minimal.yaml:35]: Also the indentation is incorrect.

Comment (NathanNam) [reply examples/stack/values-minimal.yaml:35]: Let's use this one including dropping `container_fs.*` as well.

```
metrics:
  grafana-agent:
    prom_config:
      scrape_interval: 60s
      scrape_configs:
        pod_action: drop
        cadvisor_metric_drop_regex: container_(network_tcp_usage_total|network_udp_usage_total|tasks_state|cpu_load_average_10s|fs_.*)
        cadvisor_metric_keep_regex: container_(cpu_.*|spec_.*|memory_.*|network_.*|file_descriptors)|machine_(cpu_cores|memory_bytes)
```

Comment (NathanNam) [reply examples/stack/values-minimal.yaml:35]: BTW, we should make the same change for kustomize as well.

Comment (NathanNam) [reply examples/stack/values-minimal.yaml:35]: This values-minimal.yaml is based on values-xs.yaml. What do we recommend if customers need to use a large or extra large templates?

**Comment by arthur-observe** [comment]
example only

**Pull request closed without merging**